content/news/this-week-in-fluvio-0050.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "This Week in Fluvio #50"
+date: 2023-03-31
+weight: 20
+---
+Fluvio is a distributed, programmable streaming platform written in Rust.
+
+This Week in Fluvio is our weekly newsletter for development updates to [Fluvio open source].
+
+Please subscribe to This Week in Fluvio to receive new posts in your inbox
+{{< subscribe-button >}}
+
+
+{{< banner >}}
```

Comment:
Let's remove this for now (as it will look less `marketish`) which seems to be an issue with `this week in rust`.